README.rst

```diff
@@ -68,6 +86,11 @@ Asynchronous `scroll <https://www.elastic.co/guide/en/elasticsearch/reference/cu
     loop.run_until_complete(go())
     loop.close()
 
+Running the tests
+-----------------
+
+Enter *pytest* and *tox* in project directory for run tests.
```

Comment:
```suggestion
Type *pytest* or *tox* in the project directory to run tests.
```
(still seems quite inaccurate — what does each one of those do? how do they differ?)